### PR TITLE
Docs: alignment for columns

### PR DESCRIPTION
**Pull request by PovilasKorop**
Noticed that this tweet wasn't in the docs: https://twitter.com/filamentphp/status/1673358563316584452

Sorry, first contribution, so not sure if the correct branch and section/wording for the docs.

### CODE REVIEW

**Comment by zepfietje** [comment]
Thanks for taking the time to add this, @PovilasKorop! 🙏

I think you meant to target the 2.x branch though?

**Comment by PovilasKorop** [comment]
@zepfietje no I actually meant looking into the future :) or did I miss and this feature was removed?

**Comment by zepfietje** [comment]
Oh, no the feature is definitely still there for v3.
Fine by me since v3 is pretty close anyway. 😄

File: packages/tables/docs/03-columns/01-getting-started.md [hunk 1]
```diff
@@ -386,6 +386,26 @@ TextColumn::make('title')
     ->tooltip(fn (Model $record): string => "By {$record->author->name}")
 ```
 
+## Alignment
+
+The alignment of columns can be customized by using methods like `alignRight()`:
```
Comment (zepfietje) [inline packages/tables/docs/03-columns/01-getting-started.md:391]: Table columns are aligned to the start by default. You may change the alignment using the `alignment()` method and passing it `start`, `center`, `end`, `left`, `right` or `justify`:

Comment (zepfietje) [inline packages/tables/docs/03-columns/01-getting-started.md]:
```suggestion
Table columns are aligned to the start by default. You may change the alignment using the `alignment()` method and passing it `start`, `center`, `end`, `left`, `right` or `justify`:
```

File: packages/tables/docs/03-columns/01-getting-started.md [hunk 2]
```diff
@@ -386,6 +386,26 @@ TextColumn::make('title')
...
 
+## Alignment
+
+Table columns are aligned to the start by default. You may change the alignment using the `alignment()` method and passing it `start`, `center`, `end`, `left`, `right` or `justify`:
+
+```php
+use Filament\Tables\Columns\TextColumn;
+
+TextColumn::make('name')
+    ->alignRight() // also, alignLeft(), alignJustify(), etc.
```
Comment (zepfietje) [inline packages/tables/docs/03-columns/01-getting-started.md]:
```suggestion
    ->alignment('end')
```

File: packages/tables/docs/03-columns/01-getting-started.md [hunk 3]
```diff
@@ -386,6 +386,26 @@ TextColumn::make('title')
...
+Table columns are aligned to the start by default. You may change the alignment using the `alignment()` method and passing it `start`, `center`, `end`, `left`, `right` or `justify`:
+
+```php
+use Filament\Tables\Columns\TextColumn;
+
+TextColumn::make('name')
+    ->alignment('end')
+```
+
+Alternative syntax with `alignment()`:
```
Comment (zepfietje) [inline packages/tables/docs/03-columns/01-getting-started.md]:
```suggestion
Alternatively, you may use any of the shorthand methods like `alignEnd()`:
```

File: packages/tables/docs/03-columns/01-getting-started.md [hunk 4]
```diff
@@ -386,6 +386,26 @@ TextColumn::make('title')
...
+    ->alignment('end')
+```
+
+Alternatively, you may use any of the shorthand methods like `alignEnd()`:
+
+```php
+use Filament\Tables\Columns\TextColumn;
+
+TextColumn::make('name')
+    ->alignment('right')
```
Comment (zepfietje) [inline packages/tables/docs/03-columns/01-getting-started.md]:
```suggestion
    ->alignEnd()
```

File: packages/tables/docs/03-columns/01-getting-started.md [hunk 5]
```diff
@@ -386,6 +386,26 @@ TextColumn::make('title')
...
+Table columns are aligned to the start by default. You may change the alignment using the `alignment()` method and passing it `start`, `center`, `end`, `left`, `right` or `justify`:
+
+```php
+use Filament\Tables\Columns\TextColumn;
+
+TextColumn::make('name')
+    ->alignment('end')
+```
+
+Alternatively, you may use any of the shorthand methods like `alignEnd()`:
```
Comment (zepfietje) [inline packages/tables/docs/03-columns/01-getting-started.md]:
```suggestion
Alternatively, you may use shorthand methods like `alignEnd()`:
```

**Review by zepfietje — APPROVED** [review]
I've just reorganized it a bit. Thanks again, @PovilasKorop! Nice first contribution. 😉

**Comment by PovilasKorop** [comment]
@zepfietje my pleasure! Not the last one :)

**Comment by danharrin** [comment]
Thanks!

**Pull request merged**